pages/dkp/kommander/2.2/insights/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layout.pug
+navigationTitle: DKP Insights
+title: DKP Insights
```

Comment:
I am inclined with DKP Insights but need to run it by Deepak and other key people.

Reply:
@kanna823 -we touched on that yesterday here: https://mesosphere.slack.com/archives/C02P8A08ZM3/p1648736119497329